graphql/ui_plugin_build_baron.go

```diff
@@ -115,21 +115,35 @@ func BbGetCreatedTicketsPointers(taskId string) ([]*thirdparty.JiraTicket, error
 	return results, nil
 }
 
-func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, bool, error) {
-	projectNotFoundError := false
+type buildBaronConfig struct {
+	ProjectFound     bool
```

Comment:
why is it important to break these out instead of a single boolean? (The resolver is and'ing the result anyway).

Reply:
because the service package for the old UI still needs the projectFound boolean